### PR TITLE
Chanjo2 integration howto

### LINKED ISSUE #4486: Add instructions on how to connect scout to chanjo2
Just a brief intro to chanjo2 with link and instruction to uncomment the relative parameter in the config file. Connected to #4402

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Closes #4486

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Locally, run: `mkdocs serve` and check that all links work

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by CR

### CODE REVIEW

File: docs/admin-guide/chanjo_coverage_integration.md [hunk 1]
```diff
@@ -1,11 +1,40 @@
-# Chanjo coverage integration
+# Considerations
+
+Scout has been supporting integration with the Chanjo software for several years now.
```
Comment (dnil) [inline docs/admin-guide/chanjo_coverage_integration.md:3]: Suggest either generalising to non-temporal statement, or anchoring it to particular years. Documents tend to go unedited for extended time periods, and things like "recent" only briefly is.

Comment (dnil) [inline docs/admin-guide/chanjo_coverage_integration.md:3]:
```suggestion
Scout is well integrated with the Chanjo software for coverage reporting.
```

File: docs/admin-guide/chanjo_coverage_integration.md [hunk 2]
```diff
@@ -1,11 +1,40 @@
...
+While Chanjo has been efficient and speedy in calculating metrics over transcript data, making it a valuable tool for visualizing statistics in WES analyses, the incorporation of support for other types of analyses (WGS, WTS) in Scout has rendered **Chanjo outdated**.
+
+In response to this, recent efforts at Clinical Genomics have led to the development of Chanjo2, an updated tool designed to calculate coverage and coverage completeness statistics with the advantage of utilizing the **.d4 (Dense Depth Data Dump) format**.
```
Comment (dnil) [inline docs/admin-guide/chanjo_coverage_integration.md:6]:
```suggestion
In response to this, efforts at Clinical Genomics led to the development of Chanjo2, an updated tool designed to calculate coverage and coverage completeness statistics with the advantage of utilizing the **.d4 (Dense Depth Data Dump) format**.
```

File: docs/admin-guide/chanjo_coverage_integration.md [hunk 3]
```diff
@@ -1,11 +1,40 @@
-# Chanjo coverage integration
+# Considerations
+
+Scout has been supporting integration with the Chanjo software for several years now.
+While Chanjo has been efficient and speedy in calculating metrics over transcript data, making it a valuable tool for visualizing statistics in WES analyses, the incorporation of support for other types of analyses (WGS, WTS) in Scout has rendered **Chanjo outdated**.
```
Comment (dnil) [inline docs/admin-guide/chanjo_coverage_integration.md:4]:
```suggestion
While Chanjo is efficient and speedy in calculating metrics over transcript data, making it a valuable tool for visualizing statistics in WES analyses, the incorporation of support for other types of analyses (WGS, WTS) in Scout has rendered **Chanjo outdated**. In particular, updating the reference transcript set in an active Chanjo instance is challenging.
```

File: docs/admin-guide/chanjo_coverage_integration.md [hunk 4]
```diff
@@ -1,11 +1,40 @@
...
+
+In response to this, recent efforts at Clinical Genomics have led to the development of Chanjo2, an updated tool designed to calculate coverage and coverage completeness statistics with the advantage of utilizing the **.d4 (Dense Depth Data Dump) format**.
+
+For further information on d4tools and the .d4 format, please visit the relevant [project pages][d4tools].
+Instructions on setting up a Chanjo2 server can be found in the documentation available on the [Chanjo2 GitHub pages][chanjo2].
+
+For system administrators configuring communication with either of these tools, **we recommend integrating with Chanjo2, as the Chanjo repository is likely to be deprecated in the near future**.
```
Comment (dnil) [inline docs/admin-guide/chanjo_coverage_integration.md:11]:
```suggestion
For system administrators configuring communication with either of these tools, **we recommend integrating with Chanjo2, as the Chanjo repository is likely to be deprecated**.
```

File: docs/admin-guide/chanjo_coverage_integration.md [hunk 5]
```diff
@@ -1,11 +1,40 @@
...
+
+## Chanjo coverage integration (soon deprecated)
 
 Scout may be configured to visualize coverage reports produced by [Chanjo][chanjo].
 To do so it is necessary to install the python package [Chanjo][chanjo] and its report visualization app, [chanjo-report][chanjo-report].
 
 Both these programs require a database to store samples names, transcript specifications and statistics relative to the trascript coverage for all samples. Note that chanjo can be used as a standalone but chanjo-report needs to have access to the data produced by chanjo to generate coverage reports. **Scout does not require chanjo installed in the path to visualize coverage reports, but it is necessary to install chanjo-report and set up the connection to a database containing coverage data in order to use this fuctionality**.
 
-## Chanjo database ##
+### Chanjo database
```
Comment (dnil) [inline docs/admin-guide/chanjo_coverage_integration.md:37]: 👍

**Review by dnil — APPROVED** [review]
👍 Nice! With how seldom we actually write docs, we should refrain from relative temporal statements like "recent" or "near future" though.

File: docs/admin-guide/chanjo_coverage_integration.md (hunk 1 above)
Comment (northwestwitch) [reply docs/admin-guide/chanjo_coverage_integration.md:3]: Non-temporal is good, will do!

**Pull request merged**